homeassistant/components/google_travel_time/config_flow.py

```diff
@@ -40,6 +40,20 @@
 )
 from .helpers import InvalidApiKeyException, UnknownException, validate_config_entry
 
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_MODE): vol.In(TRAVEL_MODE),
+        vol.Optional(CONF_LANGUAGE): vol.In([None, *ALL_LANGUAGES]),
+        vol.Optional(CONF_AVOID): vol.In([None, *AVOID]),
+        vol.Optional(CONF_UNITS): vol.In(UNITS),
+        vol.Optional(CONF_TIME_TYPE): vol.In(TIME_TYPES),
+        vol.Optional(CONF_TIME): cv.string,
+        vol.Optional(CONF_TRAFFIC_MODEL): vol.In([None, *TRAVEL_MODEL]),
+        vol.Optional(CONF_TRANSIT_MODE): vol.In([None, *TRANSPORT_TYPE]),
+        vol.Optional(CONF_TRANSIT_ROUTING_PREFERENCE): vol.In([None, *TRANSIT_PREFS]),
+    }
+)
+
```

Comment:
This should be modernized to use selectors instead.
I also agree with @epenet's comment